### PR TITLE
feat: add correct impact icons

### PR DESCRIPTION
**Pull request by RubenSmn**
# Description
Change icons according to design

## Type of change

- [x] New feature (non-breaking change which adds functionality)

## Screenshots
image

# Checklist:

- [x] I have performed a self-review of my own code

### CODE REVIEW

**Comment by dadiorchen** [comment]
@RubenSmn there is a failure on component test, but I thought this one is not important, we are not using it

✖  components/common/Filter.cy.js

**Comment by dadiorchen** [comment]
@RubenSmn I have solved the problem, please update your code and push again.

**Comment by RubenSmn** [comment]
@dadiorchen thanks I will. I saw this error but when testing it locally it passed. Went back in the commit history to see if anyone changed something but could not find anything

File: src/components/common/CustomCard.js [hunk 1]
```diff
@@ -44,7 +44,15 @@ function CustomCard(props) {
...
-          <SvgIcon sx={sx} component={iconURI} inheritViewBox alt="icon" />
+          <SvgIcon
+            sx={{
+              ...sx,
+              fill: 'none',
+            }}
+            component={iconURI}
+            inheritViewBox
+            alt="icon"
+          />
```
Comment (dadiorchen) [inline src/components/common/CustomCard.js:55]: Awesome, you are using the way added by @jeremydthomas

**Pull request merged**